zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -326,6 +336,46 @@ private static void initServerConnector(Server server, int port, int sslPort) {
     server.addConnector(connector);
   }
 
+  private static void runNoteOnStart(ZeppelinConfiguration conf) throws IOException, InterruptedException {
+    String noteIdToRun = conf.getNotebookRunId();
```

Comment:
It's better to add logging here to show zeppelin server is running note after started.

Reply:
Log message added.